src/test/java/no/sikt/nva/scrapers/DublinCoreValidatorTest.java

```diff
@@ -53,4 +56,23 @@ void shouldReturnWarningWhenSubjectHasUnrecognizedType() {
         assertThat(actualWarningList,
                    hasItems(new WarningDetails(Warning.SUBJECT_WARNING, List.of())));
     }
+
+    @Test
+    void shouldReturnErrorDetailsWithDoiErrorIfDoiIsInvalid() {
+        var dcValues = List.of(new DcValue(Element.IDENTIFIER, Qualifier.DOI, randomString()));
+        var dublinCore = new DublinCore(dcValues);
+        var brageLocation = new BrageLocation(Path.of("some", "ignored"));
+        var actualErrorList = DublinCoreValidator.getDublinCoreErrors(dublinCore, brageLocation);
+        assertThat(actualErrorList, hasItems(new ErrorDetails(Error.INVALID_DOI, List.of())));
+    }
+
+    @Test
+    void shouldNotReturnErrorListWithDoiIfDoiIsValid() {
+        var dcValues = List.of(new DcValue(Element.IDENTIFIER, Qualifier.DOI, "https://doi.org/10"
+                                                                              + ".5194/tc-8-1885-2014"));
+        var dublinCore = new DublinCore(dcValues);
+        var brageLocation = new BrageLocation(Path.of("some", "ignored"));
+        var actualErrorList = DublinCoreValidator.getDublinCoreErrors(dublinCore, brageLocation);
+        assertThat(actualErrorList, not(hasItems(new ErrorDetails(Error.INVALID_DOI, List.of()))));
+    }
```

Comment:
Dersom vi får inn en DublinCore med to DOI, der den ene har uri qualifier og den andre har doi qualifier, så blir den ene med i loggen som unscraped dcValue, sant?

Reply:
Fikset